hw2/server.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+import { DB } from "./fakeDb";
+import { findUser, createUser, getAutoSuggestUsers } from "./dbFunctions";
+
+const app = express();
+app.use(express.json());
+
+const PORT = process.env.PORT || 3000;
+
+app.get("/", (req, res) => {
+  res.json({ users: DB });
+});
+
+app.get("/users", (req, res) => {

```

Comment:
encapsulate this logic to users controller, "server" is typically used just for registering handler for each route, it's not supposed to contain logic in place